app-modules/ai/config/ai.php

```diff
@@ -38,5 +38,24 @@
     'supported_file_types' => [
         'application/msword',
         'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+        'text/x-c',
+        'text / x - csharp',
```

Comment:
Are the spaces suppose to be here on this mimetype? Reading the docs and the mimetype spec I think it might be `text/x-csharp`